test/org/zaproxy/zap/spider/parser/SpiderHtmlFormParserUnitTest.java

```diff
@@ -903,11 +913,12 @@ public void shouldIgnoreBaseHtmlIfActionIsAbsoluteWhenParsingPostForm() {
     @Test
     public void shouldSetValuesToFieldsWithNoValueWhenParsingGetForm() {
```

Comment:
After moving the logic from getValues method we are now failing additional JUnit tests:
- shouldSetValuesToFieldsWithNoValueWhenParsingGetForm
- shouldSetValuesToFieldsWithNoValueWhenParsingPostForm

Not sure if this was a result of removing the method or another error on our part, working to address these issues as well as modify the new test for both new parameters.